### PR TITLE
Arch Linux packaging through AUR

### PR DESCRIPTION
**Pull request by darosior**

### CODE REVIEW

**Comment by darosior** [comment]
I've tested this locally on a Docker. Don't merge until i submit the PKGBUILD to AUR (still in the process of reading the doc). See #615 for the GPG check i intentionally kept out of this first PR.

**Comment by darosior** [comment]
Hmm i need to look into adding an icon and a `.desktop` i guess..

**Comment by darosior** [comment]
I think i'll simply update the PKGBUILD to use the released .deb post v2 to install the `.desktop` and the icon.

**Comment by darosior** [comment]
https://wiki.archlinux.org/title/Namcap could be used by anyone on Arch to check our PKGBUILD.

**Comment by darosior** [comment]
To identify dependencies [find-libdeps(1)](https://man.archlinux.org/man/find-libdeps.1) (part of [devtools](https://archlinux.org/packages/?name=devtools)) can be used.

**Comment by darosior** [comment]
Updated the PKGBUILD to adhere to the [AUR packaging guidelines](https://wiki.archlinux.org/title/Arch_package_guidelines).

File: README.md [hunk 1]
```diff
@@ -56,6 +56,22 @@ as on [his Twitter profile](https://twitter.com/darosior) or his [personal
...
+```
+If you use `yay`:
+```
+yay -S liana-bin
+```
+Or manually:
+```
+git clone https://aur.archlinux.org/liana-bin.git
+cd liana-bin
+makepkg -si
```
Comment (kloaec) [inline README.md:72]: We should add the review step:
nano PKGBUILD

Comment (darosior) [reply README.md:72]: More than half the installation section is already Arch-specific... But will do.

Comment (darosior) [reply README.md:72]: Is `nano` installed by default on Arch?

**Comment by darosior** [comment]
Thanks for proofreading @kloaec.

ACK 2b3ed5a8dd4cd36a15d23f1792b37a18023f08c2

**Pull request merged**